todo_app/tests/test_view_model.py

```diff
@@ -0,0 +1,178 @@
+from todo_app.data.model import ViewModel
+from todo_app.data.item import Item
+
+import datetime
+
+
+class TestViewModel:
+
+    @staticmethod
+    def test_get_only_todo_items():
+        #Arrange
+        items = []
+        last_activity_date = datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%s")
```

Comment:
These unit tests all fail for me, because the final `%s` isn't recognised - see https://strftime.org/ for the options, but simply removing that final `.%s` appears sufficient

Note this is a really nice example of where we'd benefit from pulling the actual format string out into a variable shared by the file - so we'd only need to adjust it in one place, rather than 7

Reply:
Thanks - These changes have been made as advised